Rozdział 14/Untitled-1.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Canvas Animation</title>
+</head>
+<body>
+    <canvas id="canvas" width="200" height="200"></canvas>
+    
+    <script>
+      var canvas = document.getElementById("canvas");
+      var ctx = canvas.getContext("2d");
+
+      var size = 0;
+
+      setInterval(function () {
```

Comment:
co ja bym to zrobił to wyciągnął tą anonimową funkcję z setInterval, nazwał ją sobie jakoś ładnie i później używał jej w setInterval na zasadzie 
`setInterval(mojaWyciągnietaFunkcja, 30)`